corehq/apps/reports/filters/users.py

```diff
@@ -342,9 +344,17 @@ def pull_users_and_groups(cls, domain, mobile_user_and_group_slugs,
                 group=group,
                 simplified=True
             )
+        users_at_locations = []
+        if location_ids:
+            users_at_locations = util.get_all_users_by_domain(
+                domain=domain,
+                user_ids=user_ids_at_locations_and_descendants(location_ids),
+                simplified=True,
+                CommCareUser=CommCareUser,
+            )
```

Comment:
Oh nice!  This makes a lot of sense, as it will always be empty for reports that don't use locations